recipes/deepmei/meta.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: deepmei
-  version: 1.6.24
+  version: 1.6.27
```

Comment:
How come the new version has the same SHA256 ? I'd expect the sha256 field below to be updated too

Reply:
The source code has not changed. We received an error report from a user indicating that they could not find the specified version of TensorFlow (v2.7.0) within the default Bioconda channel. Since TensorFlow is a dependency of our tool, DeepMEI, we updated the meta.yaml file to include additional versions of TensorFlow.

Reply:
In this case there is no need to change the `version`! You should increase with 1 the `build > number` instead

Reply:
Our tools include a very large deep learning model file, which has been updated to version 1.6.27. Due to its size, we do not include this file in the Bioconda repository. Additionally, some core code is manually cloned from GitHub by the user. We have also updated the version of DeepMEI. Please merge our pull request. Thank you very much!

Reply:
I'll leave it to someone else to review it too.
To me it is very strange how a new version of a large software would reuse the old/same SHA256.

Reply:
You can find the usage instructions for our tool, DeepMEI, on GitHub: [DeepMEI on GitHub](https://github.com/xuxif/DeepMEI/tree/main). Currently, users use Conda to set up the environment and then manually clone the code and model files from GitHub. This is why the SHA256 hash has not changed. We have only modified the meta.yaml file, which is not in the tar file.

Reply:
The problem is that the recipe says version=1.6.17 but the download url is  url: https://raw.githubusercontent.com/kanglu123/deepmei/deepmei-v1.6.24/deepmei_update_v1.6.24.2.tar.gz

As I said there is no need to change the `version`. You have to bump the build number instead!